src/main.cc

```diff
@@ -8,9 +8,16 @@
 #include <string>      // std::string
 #include <string_view> // std::string_view
 #include <vector>      // std::vector
+#ifdef _WIN32
+#include <io.h>
+#define isatty _isatty
+#else
```

Comment:
Where are we actually checking for TTY in the code?

Reply:
Yes

Reply:
We haven't introduced yet . In this pr i have just inported the module for tty and color. In next pr we will introduce condition for tty as cli-program detected then we will continue using colors else we will turn off.

Reply:
I think you still need the TTY check on this one, otherwise i.e. piping the help text will result in escaped characters

Reply:
Alright @jviotti 👍